content/en/synthetics/settings/_index.md

```diff
@@ -68,19 +84,27 @@ Note: Variable values are updated whenever the test they are extracted from runs
 
 {{< /tabs >}}
 
+## Private locations
+
+Enable read and write permissions with `synthetics_private_location_read` and `synthetics_private_location_write`. Creating and modifying custom roles is an opt-in Enterprise feature. To get this feature enabled on your account, contact [Datadog support][8].
+
 ## Default settings
 
+Enable read and write permissions with `synthetics_default_settings_read` and `synthetics_default_settings_write`. Creating and modifying custom roles is an opt-in Enterprise feature. To get this feature enabled on your account, contact [Datadog support][8].
+
 ### Default locations
 
-Choose the default locations for your browser and API tests details. Options include all of the available managed locations Datadog offers and the private locations that you set up for your account.
+Choose the default locations for your API and browser tests details. Options include all of the available managed locations Datadog offers and the private locations you set up for your account.
```

Comment:
Add links to single, multistep API tests and browser tests?

Reply:
```suggestion
Choose the default locations for your [API test][10], [multistep API test ][11], or [browser test][12] details. Options include all of the available managed locations Datadog offers and the private locations you set up for your account.
```